buckaroo/Solver.fs

```diff
@@ -309,7 +318,7 @@ module Solver =
 
   let solutionCollector resolutions =
     resolutions
-    |> AsyncSeq.take 2048
+    |> AsyncSeq.take (1024 * 16)
```

Comment:
This looks so arbitrary and brittle. Why exactly does your implementation require a higher limit?
Can we compute an upper bound based on deps somehow?

Reply:
This is fairly arbitrary... I adjusted it during testing. I think having a hard limit is a good idea (yield an error sooner). However, with the new system we should fail fast on typos in branch/tag names. I think we should just leave it at a high number.